src/link_manager/LinkManagerStateMachineActiveActive.cpp

```diff
@@ -1069,7 +1074,10 @@ void ActiveActiveStateMachine::updateMuxLinkmgrState()
     Label label = Label::Unhealthy;
     if (ls(mCompositeState) == link_state::LinkState::Label::Up &&
         ps(mCompositeState) == link_prober::LinkProberState::Label::Active &&
-        (ms(mCompositeState) == mLastMuxStateNotification || mLastMuxStateNotification == mux_state::MuxState::Label::Unknown) &&
+        (ms(mCompositeState) == mLastMuxStateNotification ||
+         mLastMuxStateNotification == mux_state::MuxState::Label::Unknown ||
```

Comment:
Why can't we only check last probe state? Say if we only check last probe state: 
Case 1: Last toggle: active, last probe unknown -> healthy
Case 2: Last toggle: unknown, last probe active -> toggle should be triggered -> last toggle: active, last probe: active.

Reply:
For case 2, if the latest toggle has mux `unknown` returned, and the periodical mux probe returns `active` later, this will not trigger an extra toggle, this will be fixed in my next PR.
And we could not only check the last probe state for now because case 2 is not covered. Let's postpone this check change for a while.